go.mod

```diff
@@ -233,6 +233,7 @@ require (
 	cloud.google.com/go/iam v1.1.1 // indirect
 	cloud.google.com/go/longrunning v0.5.1 // indirect
 	code.cloudfoundry.org/gofileutils v0.0.0-20170111115228-4d0c80011a0f // indirect
+	github.com/AdaLogics/go-fuzz-headers v0.0.0-20230811130428-ced1acdcaa24 // indirect
```

Comment:
was introduced when `github.com/elastic/elastic-agent-autodiscover` was upgraded from 0.6.4 to 0.6.5 https://github.com/elastic/beats/pull/37273